Forge fails on Ubuntu Linux 16.4 LTS (System.BadImageFormatException: Could not resolve field token 0x04000008)

<!--- Provide a general summary of the issue in the Title above -->

Forge fails with the following error
```
$ forge --help

Unhandled Exception:
System.BadImageFormatException: Could not resolve field token 0x04000008
File name: 'Forge'
[ERROR] FATAL UNHANDLED EXCEPTION: System.BadImageFormatException: Could not resolve field token 0x04000008
File name: 'Forge'
```

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Install using linuxbrew, according to README.md instructions.
2. It fails right away.

## Context
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: stable 2.0.0
* Environment name and version (F#, .Net): mono 5.0.1.1
* Operating System and version: Ubuntu 16.4 LTS Desktop

Important note: It also fails if I use the mono version (5.2.x) I get with these instructions: http://www.mono-project.com/download/#download-lin

The same problem occurs if I compile from source...

Looks like `Argu.dll` isn't shipped along: <details><summary>last lines of `strace`</summary>
```
readlink("/usr", 0x7ffd9b50f6c0, 4096)  = -1 EINVAL (Invalid argument)
readlink("/usr/lib", 0x7ffd9b50f6c0, 4096) = -1 EINVAL (Invalid argument)
readlink("/usr/lib/mono", 0x7ffd9b50f6c0, 4096) = -1 EINVAL (Invalid argument)
readlink("/usr/lib/mono/gac", 0x7ffd9b50f6c0, 4096) = -1 EINVAL (Invalid argument)
readlink("/usr/lib/mono/gac/FSharp.Core", 0x7ffd9b50f6c0, 4096) = -1 EINVAL (Invalid argument)
readlink("/usr/lib/mono/gac/FSharp.Core/4.4.1.0__b03f5f7f11d50a3a", 0x7ffd9b50f6c0, 4096) = -1 EINVAL (Invalid argument)
readlink("/usr/lib/mono/gac/FSharp.Core/4.4.1.0__b03f5f7f11d50a3a/FSharp.Core.dll", 0x7ffd9b50f6c0, 4096) = -1 EINVAL (Invalid argument)
close(3)                                = 0
openat(AT_FDCWD, "/usr/lib/mono/gac/FSharp.Core/4.4.1.0__b03f5f7f11d50a3a/FSharp.Core.dll.so", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/gac/FSharp.Core/4.4.1.0__b03f5f7f11d50a3a/FSharp.Core.dll.so.la", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/aot-cache/amd64/FSharp.Core.dll.so", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/aot-cache/amd64/FSharp.Core.dll.so.la", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/gac/FSharp.Core/4.4.1.0__b03f5f7f11d50a3a/FSharp.Core.dll.config", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/etc/mono/assemblies/FSharp.Core/FSharp.Core.config", O_RDONLY) = -1 ENOENT (No such file or directory)
brk(0x55c7531b2000)                     = 0x55c7531b2000
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/Argu.dll", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/4.5//Facades/Argu.dll", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/Argu.exe", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/4.5//Facades/Argu.exe", O_RDONLY) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu/Argu.dll", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
stat("/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/bin/Argu/Argu.exe", 0x7ffd9b50fc70) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu.dll", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/Argu.dll", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/4.5//Facades/Argu.dll", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/home/develop7/.linuxbrew/Cellar/forge/2.0.0/libexec/Argu.exe", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/Argu.exe", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/lib/mono/4.5//Facades/Argu.exe", O_RDONLY) = -1 ENOENT (No such file or directory)
brk(0x55c7531d3000)                     = 0x55c7531d3000
brk(0x55c7531cf000)                     = 0x55c7531cf000
write(2, "\nUnhandled Exception:\nSystem.Bad"..., 114
Unhandled Exception:
System.BadImageFormatException: Could not resolve field token 0x04000008
File name: 'Forge'
) = 114
write(2, "[ERROR] FATAL UNHANDLED EXCEPTIO"..., 127[ERROR] FATAL UNHANDLED EXCEPTION: System.BadImageFormatException: Could not resolve field token 0x04000008
File name: 'Forge'
) = 127
```
</details>

--- a/src/Forge/app.config
+++ b/src/Forge/app.config
@@ -4,7 +4,7 @@
   <runtime>
     
   <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
-  <probing privatePath="bin" />
+  <probing privatePath="Bin" />
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />

